test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -832,7 +854,7 @@ def test_successful_fast_fund_credit_with_token_on_cft_gateway
     assert_success store
 
     options = @options.merge({ fast_fund_credit: true })
-    assert credit = @gateway.credit(@amount, store.authorization, options)
+    assert credit = @cftgateway.credit(@amount, store.authorization, options)
```

Comment:
Why did the gateway change here?

Reply:
Hmm that got in there as I was trying to investigate why all the tests were failing on the cft gateway. That test will not pass unless one or both gateways used to make the store/credit call are set to the CFT gateway, and it seems to make sense because that test appears to be trying to test the fast fund credit action on the CFT gateway using a stored PM's authorization token. After some reading, the